examples/mgui_with_threadpoolexec.py

```diff
@@ -13,28 +15,17 @@
 @magic_factory
 def make_widget(
     image: ImageData,
-    min_sigma: Annotated[float, {"min": 0.5, "max": 15, "step": 0.5}] = 4,
-    max_sigma: Annotated[float, {"min": 1, "max": 200, "step": 0.5}] = 120,
+    min_sigma: Annotated[float, {"min": 0.5, "max": 15, "step": 0.5}] = 5,
+    max_sigma: Annotated[float, {"min": 1, "max": 200, "step": 0.5}] = 30,
     num_sigma: Annotated[int, {"min": 1, "max": 20}] = 10,
     threshold: Annotated[float, {"min": 0, "max": 1000, "step": 0.1}] = 0.3,
 ) -> Future[LayerDataTuple]:
 
-    from napari.qt import thread_worker
-
-    future: Future[LayerDataTuple] = Future()
-    pbar = widgets.ProgressBar()
-    pbar.range = (0, 0)  # unknown duration
-    make_widget.insert(0, pbar)  # add progress bar to the top of widget
-
-    def _on_done(result, self=make_widget):
-        future.set_result(result)
-        self.remove(pbar)
-
     # long running function
-    @thread_worker
     def _make_blob():
+        print("running!")
```

Comment:
```suggestion
```

Reply:
since this is just an example, I left the prints in on purpose for the user?